### DISCUSSION TITLE
Is there a minikanren library that works with biwascheme?

### DISCUSSION THREAD

**Discussion by ktodyruik-ipl** [biwascheme/biwascheme#225] [Q&A]
Hi,

I was wondering if there is a minkanren implentation that works with biwascheme.

I tried the canonical implementation found here:

https://github.com/miniKanren/miniKanren

It didn't appear to work. This error was returned:

```
biwascheme> (load "mk/mk.scm")
Error: execute: unbound symbol: "e" []
Error: execute: unbound symbol: "e" [] [load]
```

**Comment by amirouche** [comment]
If you can get rid of the macros microkanren is easier to adapt to biwascheme https://github.com/jasonhemann/microKanren

Let me know if you succeed.
